tools/mtmd/mtmd.h

```diff
@@ -39,6 +39,9 @@
 #    define MTMD_API
 #endif
 
+#define MTMD_DEFAULT_MEDIA_MARKER "<__media__>"
+
+// deprecated marker, use MTMD_DEFAULT_MEDIA_MARKER instead
```

Comment:
We have such constants in `llama.h` and `ggml.h`, but we eventually have to start moving those behind API calls. It's more future-proof.

Reply:
Good idea! I added it in https://github.com/ggml-org/llama.cpp/pull/13623/commits/107790ab547604da9495c01d3be6283df3b75df6